src/components/Reactions/QuickEmojiReactions/BaseQuickEmojiReactions.js

```diff
@@ -32,9 +32,20 @@ const baseQuickEmojiReactionsPropTypes = {
     onPressOpenPicker: PropTypes.func,
 };
 
+const baseQuickEmojiReactionsDefaultProps = {
+    onEmojiSelected: () => {},
```

Comment:
I'm confused why we need to do this. These don't come from onyx right?

Reply:
To clarify, I don't think we need both `baseQuickEmojiReactionsDefaultProps` and `defaultProps` in this case, we should just have `defaultProps` for any prop that is not required

Reply:
JK, I see why it's structured like that now, my bad. Let's just remove `onEmojiSelected` from this since it's required and then we're good to go.